framework/platform/bpl/uci/cfg/bpl_cfg_uci.h

```diff
@@ -21,8 +21,11 @@
 #undef snprintf_s
 #define snprintf_s snprintf
 
-#ifndef u_int_32
-#define u_int_32 unsigned int
+#elif BEEROCKS_RDKB
```

Comment:
Please use the macros @arnout added instead of explicitly checking for BEEROCKS_RDKB

Reply:
Which macro did he add?

Reply:
```
#ifdef USE_LIBSAFEC
#define restrict __restrict
#include <libsafec/safe_str_lib.h>
#elif USE_SLIBC
#include <slibc/string.h>
#else
#error "No safe C library defined, define either USE_LIBSAFEC or USE_SLIBC"
#endif
```

Reply:
I fixed it, bu rax40 doesn't compile now.
`slibc/stdio.h: No such file or directory`